### ISSUE TITLE
Crashing when the switch keyboard appear!

### ISSUE THREAD

**Issue by butfr0g** [sudachi-emu/sudachi#175]
TOTK crashes everytime i have to name a horse is there a way to fix this? I know yuzu fixed this issue

**Comment by butfr0g** [comment]
Installing an older version of the emulator uzu 270 I found that doesn't have that keyboard it has a different method to type in the name and it works so much better.

**Comment by rosostolato** [comment]
@butfr0g can you share that apk please?

**Comment by butfr0g** [comment]
> @butfr0g can you share that apk please?

I don't know if I can share it here but you can find it on my profile.

**Comment by rosostolato** [comment]
Thank you @butfr0g. I was trying to play Hogwarts legacy but unfortunately I'm getting a black screen after the splash screen on that version.

**Comment by butfr0g** [comment]
@rosostolato weird i booted that game on that version before and isterted a name  try changing driver or setting maybe older driver

**Comment by butfr0g** [comment]
Just tried and
It boots wit turnip driver revision 13, low setting

**Comment by rosostolato** [comment]
@butfr0g What is the firmware version you're using?

**Comment by butfr0g** [comment]
@rosostolato i didn't install a firmware just the keys and it works but usually 17.00

**Comment by rosostolato** [comment]
@butfr0g did you enable async shaders?

**Comment by butfr0g** [comment]
@rosostolato yes that game is random sometimes it doesn't work for me either, did you delete the shader cache maybe that can help

**Comment by rosostolato** [comment]
Yes, I cleaned sometimes but it doesn't help. Maybe this should be a new issue on this repository.

**Comment by butfr0g** [comment]
Do you have any mods/update? Try deactivating them and see if it boots

**Comment by rosostolato** [comment]
I made it run on suyu. I first needed to run using the default driver then, after I get in, I could switch to a turnip driver. I used r18 and ran it on 720p at 30fps. The only problem is that it has so many graphic glitches that make it too ugly, not worth to play.
